ironfish/src/rpc/routes/wallet/sendTransaction.test.slow.ts

```diff
@@ -9,11 +9,14 @@ import { createRouteTest } from '../../../testUtilities/routeTest'
 import { AsyncUtils } from '../../../utils'
 
 describe('Route wallet/sendTransaction (with note selection)', () => {
-  const routeTest = createRouteTest(true)
+  const routeTest = createRouteTest()
 
   it('spends the specified notes', async () => {
     const sender = await useAccountFixture(routeTest.node.wallet, 'accountA')
 
+    routeTest.node.peerNetwork['_isReady'] = true
+    routeTest.chain.synced = true
```

Comment:
this change is the same as other tests: https://github.com/iron-fish/ironfish/blob/rahul-fixing-broken-wallet-send-test/ironfish/src/rpc/routes/wallet/createTransaction.test.slow.ts#L15